doc/development.rst

```diff
@@ -0,0 +1,214 @@
+
+Whenever an Editor position becomes vacant:
+
+* Editors announce upcoming elections and request nominations via the PEtab
+  mailing list. The time given for nominations shall be no shorter than 10 days.
+* Interested parties submit nominations including a short statement on the
+  reason for nomination before the deadline. Self-nominations are allowed.
+* Editors ask the nominees whether they accept the nomination (nominees are
+  given 5 days to accept). This step may start already during the nomination
+  phase.
+* Editors announce the nominees who accepted their nomination along with the
+  submitted statements and open the poll via the PEtab mailing list. The
+  editors choose a sensible medium and deadline for the poll.
+* The PEtab forum casts their votes secretly. (Votes may have to be open to the
+  editors, as they need to verify that only qualified votes are counted.
+  However, the editors are required to maintain confidentiality.) Every
+  participant has 1 vote per vacant position.
+* After passing the deadline, the editors count the votes and ask the
+  editor-elect to accept or decline the election. No acceptance before the end
+  of the deadline set by the editors, which shall not be less than 3 days, is
+  considered decline.
+* If an editor-elect declines, the position will be offered to the next
+  candidate according to the number of votes. (If there is no candidate left
+  with at least one vote, the election for the vacant position needs to be
+  repeated.
+* If there is a tie between candidates of which only a subset can become an
+  editor, run-off elections including only those candidates have to be
+  organized by the PEtab editors as soon as possible. Voters will again have a
+  number of votes equal to the number of vacant positions.
+* If the editor-elect accepts, the other editors announce the new editor on the
+  PEtab mailing list. The editors shall furthermore announce the number of
+  votes each nominee, elected or not, has received.
+
+Special procedure for the first election:
+-----------------------------------------
+
+The first election will be held among the authors of the original PEtab
+publication. Nominees are not required to be among the authors. The election
+is to be managed by two persons who are not among the candidates and are
+coming from two different labs. To avoid a simultaneous replacement of all
+of the editors elected during the first election, the first election will be
+subject to the following special rules: 2 persons will be elected for 3
+years, 2 persons for 2 years and one person for 1 year. The persons with
+more votes will be elected for a longer period (assuming that they accept
+for the longer period). In case of an equal number of votes among any of the
+top 5 candidates, there will be run-off elections between those candidates
+with equal numbers of votes. The editors-elect are given 3 working days to
+accept the election.
+
```

Comment:
In the email announcing the editors we had the phrase: "(1 year, with the option of +1 year if one of the above editors hands over his responsibility)"
Should we include this in the document? Something like
```suggestion
If an editor decides to hand over his editorial role before his election period expires, editors elected for a shorter time period can decide to take over and expand their time as an editor for the remaining time.
```